### PR TITLE
Chore: Added a question for unstaging files using git

### PR DESCRIPTION
**Pull request by lakhanjindam**
Checklist:

 - [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org/).
 - [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (not a vague title like Update index.md)

Closes #XXXXX
Git question about unstaging a staged file using git cli

### CODE REVIEW

**Comment by lakhanjindam** [comment]
@jdwilkin4 please review the changes

File: src/data/git-quiz.ts [hunk 1]
```diff
@@ -533,6 +533,16 @@ const gitQuiz = [
     Explanation:
       "git commit --amend  is the command used to change the latest commit message.",
     Link: "https://forum.freecodecamp.org/t/git-guide-how-to-amend-your-most-recent-git-commit-message/13186"
+  },
+  {
+    Question: "How to unstage a staged file in git ?",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:538]:
```suggestion
    Question: "How do you unstage a staged file in git?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by lakhanjindam** [comment]
> Thank you for your contribution!
> 
> Please review the requested changes. Once those changes have been implemented, then we can merge your PR into main. 👍

Done

**Pull request merged**